up & going/ch1.md

```diff
@@ -3,7 +3,7 @@
 
 Bem vindo à série *You Don't Know JS* (*YDKJS*).
 
-*Iniciando* é uma introdução à diversos conceitos básicos de programação -- claro que inclinado ao uso do JavaScript (muitas vezes abreviado como JS) especificamente -- e como abordar e entender o resto dos títulos nessa série. Especialmente se você está começando em programação ou JavaScript, esse livro irá, de forma breve, explorar tudo que você precisa saber para *iniciar*.
+*Iniciando* é uma introdução à diversos conceitos básicos de programação -- claro que iniciando ao uso do JavaScript (muitas vezes abreviado como JS) especificamente -- e como abordar e entender o resto dos títulos nessa série. Especialmente se você está começando em programação ou JavaScript, esse livro irá, de forma breve, explorar tudo que você precisa saber para *iniciar*.
```

Comment:
Olá @wallacyVerOne,

A ideia da frase original é que veremos os conceitos básicos de programação, inclinados ao uso JavaScript:

> [...] of course _we lean toward_ JavaScript (often abbreviated JS) specifically [...]

Pode conferir no original:
https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/up%20%26%20going/ch1.md

De qualquer forma, obrigado 😉